governance/metagov.py

```
@@ -0,0 +1,185 @@
+              f"全仓 {len(anchors)} job；pending {pend} 诚实不造数）")
+        return 0
+
+    if cmd == "promote":
+        reg = sys.argv[sys.argv.index("--registry") + 1]
+        rec_f = sys.argv[sys.argv.index("--record") + 1]
+        try:
+            rec = json.loads(Path(rec_f).read_text(encoding="utf-8"))
+        except (OSError, json.JSONDecodeError) as e:
+            die2(f"record 不可读: {e}")
+        if {"practice", "goal", "evidence", "promoted_by"} - set(rec):
+            die2("晋级记录缺必填（practice/goal/evidence/promoted_by）")
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)